x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/elastic/authorization/ElasticInferenceServiceAuthorizationHandlerTests.java

```diff
@@ -1,283 +0,0 @@
-/*
```

Comment:
Class under test was removed so we don't need it anymore. Most of the functionality was moved to `AuthorizationPoller` and its test files.